src/main/resources/avro/readmethods.avdl

```diff
@@ -14,8 +14,10 @@ to `ReferenceSet`s containing that same `Reference`. If no reference is
 specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
 */
 record SearchReadsRequest {
-  /** If nonempty, restrict this query to reads within the given readgroups. */
-  array<string> readGroupIds = [];
+  /**
+  The ReadGroup to search.
+  */
+  string readGroupId;
```

Comment:
I am very hesitant to push complexity down to the client.  If that is the case, why even have the server?  I am more in favor of `array<string> readGroupSetIds`, since in the future if we want to drill back to the reads after performing  Matchmaker or Beacon, this is exactly the case that would be needed where it would be across all repositories of stored reads.